tox.ini

```diff
@@ -1,57 +1,92 @@
 [tox]
 envlist =
-    py27,py35,py36,py37,py38,pypy
-    docs,{py2,py3}-cover,coverage,
+    lint,
+    py27,pypy,
+    py35,py36,py37,py38,pypy3,
+    docs,
+    coverage
+isolated_build = True
 
 [testenv]
-# Most of these are defaults but if you specify any you can't fall back
-# to defaults for others.
-basepython =
-    py27: python2.7
-    py35: python3.5
-    py36: python3.6
-    py37: python3.7
-    py38: python3.8
-    pypy: pypy
-    pypy3: pypy3
-    py2: python2.7
-    py3: python3.5
+commands =
+    python --version
+    pytest {posargs:}
+extras =
+    testing
+setenv =
+    COVERAGE_FILE=.coverage.{envname}
+
+[testenv:coverage]
+basepython = python3.8
+commands =
+    coverage combine
+    coverage xml
+    coverage report --show-missing
```

Comment:
show-missing is already set in coveragerc... I do think this needs a `--fail-under 100` though

Reply:
I think skip_install=True can be set on this target as well

also on other tox files I think we've been able to get rid of basepython for the most part

Reply:
removing `basepython` is fine by me, just means that we end up using whichever python version `tox` is installed under, instead of the specific one mentioned.

Reply:
In most cases this is still going to be Python 2.7... even on the Github `ubuntu-latest` runners `python` is still Python 2.7.

Reply:
Hmmm, actually, never mind, it looks like whatever Python was installed last is first in the PATH, so as long as I install Python 2.7 followed by Python 3.x it works fine to have `coverage` env run under Python 3.8.